<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Git::Internal::BranchDialog</class>
 <widget class="QDialog" name="Git::Internal::BranchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Branches</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="infoGroupBox">
     <property name="title">
      <string>General information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="repositoryLabel">
        <property name="text">
         <string>Repository:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="repositoryFieldLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="localBranchGroupBox">
     <property name="title">
      <string>Branches</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="localBranchListView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="remoteBranchGroupBox">
     <property name="title">
      <string>Remote branches</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="remoteBranchListView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Git::Internal::BranchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
